mantidimaging/gui/windows/live_viewer/view.py

```diff
@@ -35,6 +36,33 @@ def __init__(self, main_window: 'MainWindowView', live_dir_path: Path) -> None:
         self.imageLayout.addWidget(self.live_viewer)
         self.live_viewer.z_slider.valueChanged.connect(self.presenter.select_image)
 
+        self.activated_operations: Set[str] = set()
+        self.filter_params: Dict[str, Dict] = {}
```

Comment:
Can these be combined into just a dictionary. So when a filter is turned off we removed the entry in the dictionary.
```
del self.filter_params["Rotate Stack"]
```